src/pages/NewChatPage.tsx

```diff
@@ -218,7 +232,8 @@ function NewChatPage({betas, isGroupChat, personalDetails, reports, translate, i
             if (!interactionTask) {
                 return;
             }
-            interactionTask.cancel();
+
+            (interactionTask as {cancel: () => void}).cancel();
```

Comment:
I think `doInteractionTask` is wrongly typed becuase `cancel` method will be only there for `desktop` platform maybe we can improve typing there?
CC: @fabioh8010 @blazejkustra

Reply:
I agree, @ruben-rebelo let's fix it here. Here is the working code to adjust:

src/libs/DoInteractionTask/types.ts

```ts
import type {InteractionManager} from 'react-native';

type DoInteractionTask = (callback: () => void) => ReturnType<typeof InteractionManager.runAfterInteractions> | null;

export default DoInteractionTask;
```

src/libs/DoInteractionTask/index.ts

```ts
import type DoInteractionTask from './types';

const doInteractionTask: DoInteractionTask = (callback) => {
    callback();
    return null;
};

export default doInteractionTask;
```

src/libs/DoInteractionTask/index.desktop.ts

```ts
import {InteractionManager} from 'react-native';
import type DoInteractionTask from './types';

// For desktop, we should call the callback after all interactions to prevent freezing. See more detail in https://github.com/Expensify/App/issues/28916
const doInteractionTask: DoInteractionTask = (callback) =>
    InteractionManager.runAfterInteractions(() => {
        callback();
    });

export default doInteractionTask;
```